packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -60,10 +72,35 @@ export const Webchat = forwardRef((props, ref) => {
     openWebviewT,
     closeWebviewT
   } = props.webchatHooks || useWebchat()
+  const { theme } = webchatState
   const { initialSession, initialDevSettings, onStateChange } = props
   const [botonicState, saveState, deleteState] = useLocalStorage('botonicState')
-  const [menuIsOpened, setMenuIsOpened] = useState(false)
-  const [emojiIsOpened, setEmojiIsOpened] = useState(false)
+  const [persistentMenuOpened, setPersistentMenuOpened] = useState(false)
+  const [emojiPickerOpened, setEmojiPickerOpened] = useState(false)
+  const [attachment, setAttachment] = useState({})
+
+  const useTheme = property => {
```

Comment:
I know I gave you the idea... but I don't think `useTheme` is a good name for this function as it's not a hook. It could cause confusion in the future...

Reply:
renamed to `getThemeProperty` as is what is doing in fact, i'm agree with you 👍